src/Observables.jl

```diff
@@ -64,19 +107,12 @@ end
 Base.convert(::Type{T}, x::T) where {T<:Observable} = x  # resolves ambiguity with convert(::Type{T}, x::T) in base/essentials.jl
 Base.convert(::Type{T}, x) where {T<:Observable} = T(x)
 
-function Base.getproperty(obs::Observable, field::Symbol)
-    if field === :val
-        return getfield(obs, field)
-    elseif field === :listeners
-        return getfield(obs, field)
-    elseif field === :inputs
-        return getfield(obs, field)
-    elseif field === :id
-        return obsid(obs)
-    else
-        error("Field $(field) not found")
-    end
+struct Consume
+    x::Bool
 end
+Consume() = Consume(true)
+Consume(x::Consume) = Consume(x.x) # for safety in selection_point etc
+Base.:(==)(a::Consume, b::Consume) = a.x == b.x
```

Comment:
Do you need this method at all? Isn't that the fallback behavior? (It's not `mutable`.)